downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/README.md

```diff
@@ -48,4 +44,7 @@ UniProtKB       A0AVI4  TM129_HUMAN             GO:0061630      PMID:24807418
 UniProtKB       A5YM72  CRNS1_HUMAN             GO:0047730      REACTOME:R-HSA-6786245  TAS             F                       protein taxon:9606      20150706        Reactome
 ```
 
-This shows different annotation types that can be found in `gene_association.reactome`. {TODO: Describe the N/As and some of the repetitive column values}
+This shows different annotation types that can be found in `gene_association.reactome`. There are four columns (marked with N/A) that are not populated by the GOA prepare step. In GO Annotation format, these would be the Qualifier (4th column), With (or) from (8th column), DB Object Name (10th), and DB Object Synonym (11th) columns.
+
+In this sample file you can also see the annotation formats for each GO annotation type. Molecular Function annotations (lines 4 and 5) can be of two forms. The first, at line 4, are for Molecular Function annotations where an existing PubMed literature reference existed. These receive the 'PMID' prefix on their EventIdentifier and the EvidenceCode 'EXP', meaning it was inferred from an experiment (<a href="http://geneontology.org/docs/guide-go-evidence-codes/">source</a>). The second, at line 5, are those without a PubMed reference. These receiver the typical 'REACTOME' prefix for their EventIdentifier and 'TAS' for the evidence code (Traceable Author Statement). 
```

Comment:
The link for the evidence code doc might be better on the "EvidenceCode" text just before 'EXP' rather than on the word "source".